test/Microsoft.TestPlatform.Utilities.UnitTests/CodeCoverageDataAttachmentsHandlerTests.cs

```diff
@@ -50,7 +50,6 @@ public CodeCoverageDataAttachmentsHandlerTests()
 #endif
     }
 
-#if NETFRAMEWORK
```

Comment:
Before v3 the current directory was bin which wasn't uniform with netfx, this is now standard so we also need to copy the files.